### PR TITLE
Label selector retrieval tests

### PR DESCRIPTION
**Pull request by nebril**
Slight refactor of deploy command to make it testable

---
This change is [Reviewable](https://reviewable.io/reviews/mirantis/k8s-appcontroller/89)

### CODE REVIEW

**Comment by pigmej** [comment]
Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/k8s-appcontroller/89)*

File: cmd/deploy.go [hunk 1]
```diff
@@ -93,10 +91,39 @@ func deploy(cmd *cobra.Command, args []string) {
...
-	Run:   deploy,
+func getLabelSelector(cmd *cobra.Command) (string, error) {
+	labelSelector, err := cmd.Flags().GetString("label")
+	if labelSelector == "" {
+		labelSelector = os.Getenv("APPCONTROLLER_LABEL_SELECTOR")
+	}
+	return labelSelector, err
+}
+
+// InitRunCommandreturns cobra command for performing AppController graph deployment
```
Comment (jellonek) [inline cmd/deploy.go]: missing space

**Comment by nebril** [comment]
Review status: 2 of 3 files reviewed at latest revision, 1 unresolved discussion.

---

*[cmd/deploy.go, line 102 at r1](https://reviewable.io:443/reviews/mirantis/k8s-appcontroller/89#-KSfxKm7Wj1YvB0oF2J9-r1-102:-KSfzC65dZwry2Kmirgv:b-896fix) ([raw file](https://github.com/mirantis/k8s-appcontroller/blob/ac11105fbfa6ed328186c87465f810ae295eb18f/cmd/deploy.go#L102)):*
<details><summary><i>Previously, jellonek (Piotr Skamruk) wrote…</i></summary>
> missing space
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/k8s-appcontroller/89)*

**Comment by pigmej** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[cmd/deploy.go, line 102 at r1](https://reviewable.io:443/reviews/mirantis/k8s-appcontroller/89#-KSfxKm7Wj1YvB0oF2J9-r1-102:-KSfzSrQErNQtUBe_TjQ:b32w0x) ([raw file](https://github.com/mirantis/k8s-appcontroller/blob/ac11105fbfa6ed328186c87465f810ae295eb18f/cmd/deploy.go#L102)):*
<details><summary><i>Previously, nebril (Maciej Kwiek) wrote…</i></summary>
> Done.
</details>
OK

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/k8s-appcontroller/89)*

**Comment by pigmej** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/k8s-appcontroller/89#-:-KSfzdCfFehaAs700AaK:bnfp4nl)*

**Comment by jellonek** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/k8s-appcontroller/89#-:-KSg0i9Us0rHsNwFJ4FL:bnfp4nl)*

**Pull request merged**